app.js

```diff
@@ -1,48 +1,64 @@
+import Book from './book.js';
+
 let Books = JSON.parse(localStorage.getItem('Books')) ?? [];
 
-function removeBook(id) {
-  const book = document.getElementById(id);
-  book.remove();
-  Books = Books.filter((bookObj) => bookObj.id !== id);
-  localStorage.setItem('Books', JSON.stringify(Books));
-}
+class Library {
+  constructor(book) {
+    this.book = book;
+  }
 
-function displayBook(id, title, author) {
-  const bookList = document.getElementById('book-list');
-  const li = document.createElement('li');
-  li.setAttribute('id', id);
-  li.innerHTML = `
-      <h3> ${title} </h3> 
-      <p>  ${author} </p>
-      `;
-  const removeBtn = document.createElement('button');
-  removeBtn.innerHTML = 'Remove';
-  removeBtn.addEventListener('click', () => removeBook(id));
-  li.appendChild(removeBtn);
-  bookList.appendChild(li);
-}
+  addBook() {
+    Books.push(this.book);
+    localStorage.setItem('Books', JSON.stringify(Books));
+  }
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a feature to your form which clears the input fields once a user has added a form. This will make your application behaviors professionally and will prevent users to add one book multiple times. To help you, you can refer this **[article](https://www.w3schools.com/jsref/met_form_reset.asp)**

Reply:
Thank you for your advice, it's great advice 👍🏼